src/metax_api/api/rest/base/views/common_view.py

```diff
@@ -196,6 +196,9 @@ def get_queryset(self):
         else:
             queryset = queryset.select_related(*self.select_related)
 
+        # why we are not returning not found here??
+        # if not queryset:
+        #     raise Http404
```

Comment:
This comment is here because noticed different behavior for differently queried empty querysets. 
More pondering below.